### ISSUE TITLE
Dual Language Mode for Auto-Correction

### ISSUE THREAD

**Issue by 0xSegmented** [futo-org/android-keyboard#708]
Hi Futo,

The Gboard has the function to use multiple languages at the same time. For example, the auto completion would show both German and English words.

Would be great if this gets assigned, plus it ditches the switching languages.

**Comment by dean-futo** [comment]
#708 is a duplicate - merging that issue into this one.

**Comment by Chinoman10** [comment]
Isn't this the same as
- #461 ?

Except maybe 461 is broader and this one is specific to just 2 languages at once?

**Comment by dean-futo** [comment]
@Chinoman10 yes, they are essentially the same. Some languages are easier to implement than others so this will be a gradual rollout.

**Comment by abb128** [comment]
Closing as duplicate of #683
